src/openqaoa-qiskit/backends/qaoa_qiskit_qpu.py

```diff
@@ -219,7 +240,8 @@ def get_counts(self, params: QAOAVariationalBaseParams, n_shots=None) -> dict:
                     job_state = True
                     no_of_api_retries += 1
                     time.sleep(5)
-                except (IBMQJobFailureError, IBMQJobInvalidStateError):
+                except (IBMQJobFailureError, IBMQJobInvalidStateError) as e:
+                    raise(e)
```

Comment:
If you raise here, the print wont happen.

Reply:
fair point, removed the raising